src/bundle.ts

```diff
@@ -67,20 +62,24 @@ export class BuildBundler extends Transform {
     const forkedAnalyzer = analyzer ? analyzer._fork({urlLoader}) :
                                       buildAnalyzer.analyzer._fork({urlLoader});
 
+    strategy = strategy ||
```

Comment:
The strategy is now a constructor argument for the Bundler.  It was previously handed to generateManifest.  Was that your question?